### DISCUSSION TITLE
Azure AI Studio content filter errors

### DISCUSSION THREAD

**Discussion by AAnirudh07** [BerriAI/litellm#5831] [Q&A] [answered]
Hello team,
If an Azure AI Studio endpoint triggers a content filter error, is it handled using `litellm.exceptions.ContentPolicyViolationError`?

Thanks!

**Comment by ishaan-jaff** [comment, marked as answer]
yes @AAnirudh07  - that's expected behavior

**Comment by AAnirudh07** [comment]
thanks @ishaan-jaff
